opencti-platform/opencti-graphql/src/domain/group.js

```diff
@@ -54,7 +54,20 @@ export const batchRoles = async (context, user, groupIds) => {
 };
 
 export const addGroup = async (context, user, group) => {
-  const created = await createEntity(context, user, group, ENTITY_TYPE_GROUP);
+  let groupWithDefaultValues = group;
+  if (groupWithDefaultValues.default_assignation === undefined) {
+    groupWithDefaultValues = {
+      ...groupWithDefaultValues,
+      default_assignation: false,
+    };
+  }
+  if (groupWithDefaultValues.auto_new_marking === undefined) {
+    groupWithDefaultValues = {
+      ...groupWithDefaultValues,
+      auto_new_marking: false,
+    };
+  }
```

Comment:
I think it could be simplified, maybe like this :
```
const groupWithDefaultValues = {
    ...group,
    default_assignation: group.default_assignation ?? false,
    auto_new_marking: group.auto_new_marking ?? false,
  };
```
What do you think @Archidoit ?

Reply:
yes you're right!! I'll modify this!

Reply:
--> done by @richard-julien